pkgs/tools/admin/aws-rotate-key/default.nix

```diff
@@ -2,15 +2,15 @@
 
 buildGoPackage rec {
   name = "aws-rotate-key-${version}";
-  version = "1.0.0";
+  version = "1.0.2";
 
   goPackagePath = "github.com/Fullscreen/aws-rotate-key";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "Fullscreen";
     repo = "aws-rotate-key";
-    sha256 = "13q7rns65cj8b4i0s75dbswijpra9z74b462zribwfjdm29by5k1";
+    sha256 = "0pcdnp1i88wxbxjj9cwpxasa785wc1nd6iy54sb12i95alxc4r3ml";
```

Comment:
you seem to have an extra `l` at the end of the sha256, the correct sha is `0pcdnp1i88wxbxjj9cwpxasa785wc1nd6iy54sb12i95alxc4r3m`

Reply:
whoops, sorry about that!